api/helper/authenticate_user.go

```diff
@@ -6,13 +6,29 @@ import (
 	"userland/store"
 )
 
-func AuthenticateUser(r *http.Request, tokenStore store.TokenStore) (string, error) {
-	tokenAuth, err := jwt.ExtractTokenMetadata(r)
+func AuthenticateUserAccessToken(r *http.Request, tokenStore store.TokenStore) (string, error) {
+	tokenAuth, err := jwt.ExtractAccessTokenMetadata(r)
 	if err != nil {
 		return "", err
 	}
 
-	userId, err := jwt.FetchAuth(tokenAuth, tokenStore)
+	//IsAtStillActive
```

Comment:
Sorry, I might have missed this one in the previous review. What does this comment mean? 😅 
Similar question for the `//IsRtStillActive`